website/content/en/docs/deployment/prerequisites.md

```diff
@@ -1,63 +1,80 @@
+
+For more information about DLAMI options, see [Release Notes for DLAMI](https://docs.aws.amazon.com/dlami/latest/devguide/appendix-ami-release-notes.html). For information on instance type selection, see [Selecting the Instance Type for DLAMI](https://docs.aws.amazon.com/dlami/latest/devguide/instance-select.html).
+
+### Option 2: Docker
+
+Create a Ubuntu environment using Docker. Pull the latest version of the Ubuntu image of your choice. For more information, see [Pull an image by digest (immutable identifier)](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier) in the Docker documentation.
+```sh
+docker pull ubuntu:18.04
 ```
 
-Run the following command to create an EKS cluster:
-```bash
-eksctl create cluster \
---name ${CLUSTER_NAME} \
---version 1.21 \
---region ${CLUSTER_REGION} \
---nodegroup-name linux-nodes \
---node-type m5.xlarge \
---nodes 5 \
---nodes-min 5 \
---nodes-max 10 \
---managed \
---with-oidc
+Connect to localhost from your container:
+```sh
+docker container run -it -p 127.0.0.1:8080:8080 ubuntu:18.04
 ```
 
-If you are using an existing EKS cluster, create an [OIDC provider](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) and associate it with for your EKS cluster with the following command:
-```bash
-eksctl utils associate-iam-oidc-provider --cluster ${CLUSTER_NAME} \
---region ${CLUSTER_REGION} --approve
+Download the latest package information: 
+```sh
+apt update
+```
+
+ Install the necessary tools: 
+```sh
+apt install git curl unzip tar make sudo vim wget -y
 ```
-More details about cluster creation via `eksctl` can be found in the [Creating and managing clusters](https://eksctl.io/usage/creating-and-managing-clusters/) guide.
 
-## Clone the repository 
+### Option 3: AWS Cloud9
+
+Launch a Ubuntu instance using Cloud9:
+```sh
+Launch Ubuntu 18.04 cloud9 instance 
+```
+
+## Clone repository 
+
 Clone the [`awslabs/kubeflow-manifests`](https://github.com/awslabs/kubeflow-manifests) and the [`kubeflow/manifests`](https://github.com/kubeflow/manifests) repositories and check out the release branches of your choosing.
 
-Substitute the value for `KUBEFLOW_RELEASE_VERSION`(e.g. v1.6.0) and `AWS_RELEASE_VERSION`(e.g. v1.5.1-aws-b1.0.1) with the tag or branch you want to use below. Read more about [releases and versioning]({{< ref "/docs/about/releases.md" >}}) if you are unsure about what these values should be.
+Substitute the value for `KUBEFLOW_RELEASE_VERSION`(e.g. v1.6.0) and `AWS_RELEASE_VERSION`(e.g. v1.6.1-aws-b1.0.1) with the tag or branch you want to use below. Read more about [releases and versioning]({{< ref "/docs/about/releases.md" >}}) if you are unsure about what these values should be.
```

Comment:
v1.6.1-aws-b1.0.1 will probably not be our release version the x.x.1 indicated a patch release for a bug fix. -b1.0.0 is more likely to be our release version @surajkota is this accurate ? Idk if I've missed something.